src/elements/content-sidebar/activity-feed/task-form/TaskForm.scss

```diff
@@ -53,6 +53,17 @@
             .pill-selector-input-wrapper {
                 width: auto;
             }
+
+            .overlay-wrapper {
+                border: 1px solid $sf-fog;
```

Comment:
does this gray map to any of the `bdl-neutral` colors?

Reply:
nope, and it seems a lot of our borders in general end up using this `sf-fog` color as well